python/lsst/ts/tunablelaser/mock_server.py

```diff
@@ -29,10 +35,37 @@
 
 from .enums import Mode, NoSCU, Output, Power, SCUConfiguration
 
-TERMINATOR = "\r\n\x03"
+TERMINATOR = b"\r\n\x03"
 
 
-class MockServer(tcpip.OneClientServer):
+class StubbsLaserServer(tcpip.OneClientReadLoopServer):
+    """Implement Stubbs mock server.
+
+    Attributes
+    ----------
+    device : `MockNT252`
+        The mock NT252 device.
+    """
+
+    def __init__(self) -> None:
+        self.device = MockNT252()
+        super().__init__(
+            port=9001,
```

Comment:
Please do not hardcode the values for port, host, encoding and terminator but take them from configuration.